### PR TITLE
Send email after delay. 

### PR DESCRIPTION
**Pull request by aksanoble**

### CODE REVIEW

File: src/components/Toast.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    <Container>
+      <span>Sending...</span>
+      <button onClick={handleUndo}>Undo</button>
+    </Container>
+  )
+}
+
+const Container = styled.div`
+  position: fixed;
+  left: 200px;
```
Comment (AquiGorka) [inline src/components/Toast.js]: No need to update this, but we should a) double check this is the correct x-offset with Shokunin and b) if it is get the value from the `SIDEBAR_WIDTH` constant in the other file

File: src/components/Toast.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  bottom: 50px;
+  height: 60px;
```
Comment (AquiGorka) [inline src/components/Toast.js]: My suggestion: don't set hardcoded heights, try using padding to get that extra space.

File: src/components/Composer.js [hunk 3]
```diff
@@ -59,13 +60,26 @@ function ExpandedComposer() {
...
+    setSendToastVisible(false)
+  }
   const handleSubmit = ev => {
     ev.preventDefault()
+    setSendToastVisible(true)
+  }
+
+  const onToastTimeout = () => {
+    setSendToastVisible(false)
+    setTo('')
```
Comment (AquiGorka) [inline src/components/Composer.js]: How about we do the cleanup after we execute the `sendMsg` function to avoid weird race conditions

Comment (aksanoble) [reply src/components/Composer.js]: Done.

File: src/components/Toast.js (hunk 2 above)
Comment (aksanoble) [reply src/components/Toast.js]: Done.

**Pull request merged**